### PR TITLE
Jensupdate

### PR DESCRIPTION
**Pull request by Aiglobelam**
I want to merge this readme update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # gittest
+
```
Comment (joelbeats) [inline README.md:2]: This looks awsome / joel

Comment (Aiglobelam) [reply README.md:2]: 👍

Comment (joelbeats) [reply README.md:2]: 💯

**Review by joelbeats — APPROVED** [review]
Jättebra

**Pull request merged**